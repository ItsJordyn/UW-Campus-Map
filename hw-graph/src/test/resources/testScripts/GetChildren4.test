# GetChildren4 testing file
# Tests that the children nodes from a single node can be
# Printed out (Node has an edge pointing to itself).

# Create a graph
CreateGraph graph1

# Add three nodes
AddNode graph1 n1

#Add edges
AddEdge graph1 n1 n1 e1

# Print the nodes and the outgoing edges from n1
ListChildren graph1 n1