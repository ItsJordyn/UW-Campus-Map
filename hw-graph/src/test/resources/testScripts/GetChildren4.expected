# GetChildren4 testing file
# Tests that the children nodes from a single node can be
# Printed out (Node has an edge pointing to itself).

# Create a graph
created graph graph1

# Add three nodes
added node n1 to graph1

#Add edges
added edge e1 from n1 to n1 in graph1

# Print the nodes and the outgoing edges from n1
the children of n1 in graph1 are: n1(e1)